jep/11/README.adoc

```diff
@@ -76,10 +75,8 @@ For a user who is interested in adopting a plugin:
    link:https://github.com/jenkins-infra/repository-permissions-updater to request permission to deploy snapshots and releases of the plugin.
 
 For a maintainer who is interested in giving up maintainership of a plugin:
-1. The maintainer should edit the plugin's wiki page and add the *adopt-this-plugin* label.
-2. Alternatively, the maintainer can use the following JIRA macro on the plugin's wiki page:
 
- {jenkins-plugin-info:myplugin|adopt-message=The maintainer is looking for a co-maintainer.}
+1. The maintainer should edit the plugin's GitHub repo page and add the *adopt-this-plugin* label.
```

Comment:
Not useful to have a list with one entry.

Reply:
Right, but I have a another fix suggestion.

@daniel-beck @rodrigc Since we are now in the documentation-as-code world, I suggest to replace this entire section by a link to https://github.com/jenkins-infra/jenkins.io/blob/988f8520255f59c218d0c6ea9d1ae12991b7a9bb/content/doc/developer/plugin-governance/adopt-a-plugin.adoc (pinned version of the current guide). If you agree, I will do it in a follow-up PR

Reply:
ping @rodrigc

Reply:
Ok that sounds fine

Reply:
Seems weird to me as it inverts the relationship of JEPs with their implementation.

Reply:
This is a historical implementation @daniel-beck ("before"), not the suggested one